pkg/apis/serving/v1alpha3/framework_kfserver.go

```diff
@@ -0,0 +1,28 @@
+package v1alpha3
+
+import v1 "k8s.io/api/core/v1"
+
+// KFServerSpec defines arguments for configuring PyTorch model serving.
```

Comment:
Where do sklearn/xgboost go now?

Reply:
They live inside the KFServer. The proposal discusses a fatter inference server to support and dynamically detect these frameworks.